single.php

```diff
@@ -1,32 +1,50 @@
 <?php
```

Comment:
This template changed to match the streamlined structure of our other templates, it included lots of code that we no longer used, which I removed. Also, his template is mostly used on Child sites as the main site doesn't have many posts - if any.